docker/service/src/main/java/uk/nhs/adaptors/scr/clients/spine/SandboxSpineClient.java

```diff
@@ -29,31 +31,37 @@
         return new Response(OK.value(), null, getResourceAsXmlDocument(ACS_SET_PERMISSION_RESPONSE));
     }
 
+    @SneakyThrows
     @Override
     public Response<String> sendScrData(String requestBody, String nhsdAsid, String nhsdIdentity, String nhsdSessionUrid) {
         Header[] headers = {
             new BasicHeader(CONTENT_LOCATION, ""),
-            new BasicHeader(RETRY_AFTER, "300")
+            new BasicHeader(RETRY_AFTER, "100")
         };
+        sleep(200);
```

Comment:
Remove repeated integers and add a global variable that we can set at the docker container level for quick modifications and deployment